# Data to be logged

Header header
string type
float32 data
